backend_plugin/src/evaluators/EvaluatorManager.java

```diff
@@ -94,4 +72,41 @@ public void addEvaluator(IEditorPart editorWindow) {
 	public void notifyFeatureSuggestion(String featureID) {
 		this.fs.notifyAllObservers(featureID);
 	}
+
+	public void start() {
+		for (IWorkbenchWindow window : PlatformUI.getWorkbench().getWorkbenchWindows()) {
```

Comment:
Is it possible to add some inline comments on some of this? I can kind of guess what is going on, but having something short like "Grab all open documents" "Look at all windows in Eclipse", something that gives just a bit more context